packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -284,17 +292,28 @@ class FloatingActionButton extends StatelessWidget {
 
     result = RawMaterialButton(
       onPressed: onPressed,
-      elevation: elevation,
-      highlightElevation: highlightElevation,
-      disabledElevation: disabledElevation,
+      elevation: elevation
+        ?? floatingActionButtonTheme.elevation
+        ?? _defaultElevation,
+      highlightElevation: highlightElevation
+        ?? floatingActionButtonTheme.highlightElevation
+        ?? _defaultHighlightElevation,
+      disabledElevation: disabledElevation
```

Comment:
If you pull these into variables like I mentioned above, then this one can be simplified to just `disabledElevation ?? elevation`

Reply:
Done